README.md

```diff
@@ -62,6 +62,7 @@ Confused on how this works, or you want to see more? Checkout the [examples](/ex
 | **portCache**              | boolean | true      | After you queried a server, the second time you query that exact server (identified by specified ip and port), first add an attempt to query with the last successful port.                                                   |
 | **noBreadthOrder**         | boolean | false     | Enable the behaviour of retrying an attempt X times followed by the next attempt X times, otherwise try attempt A, then B, then A, then B until reaching the X retry count of each.                                           |
 | **checkOldIDs**            | boolean | false     | Also checks the old ids amongst the current ones.                                                                                                                                                                             |
+| **rejectUnauthorized**     | boolean | true      | Specifies whether connections using invalid certificates (i.e. self-signed certificates) are rejected.                                                                                                                        |
```

Comment:
Games that require additional arguments are not to be listed here, only in the games list file.

Reply:
Removed commentary from README.md but left in place in GAMES_LIST.md.